### PR TITLE
Mini challenge 1 and 2

### PR DESCRIPTION
**Pull request by vmcruz**
## Description
This PR covers challenges 1 and 2. Thank you.

### Using create-react-app, what do we need to set up for testing?
The setupTests.js file at the root of the project, along with react-testing-library for testing the components in isolation

### What components are worth to test in your development?
Those where the user has the more interactions with.

### Can you apply TDD once you already created components?
Yes, but TDD is based on the idea that the test are designed first and then do the development.

### CODE REVIEW

File: src/components/Button/Button.component.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  icon: PropTypes.string,
+  color: PropTypes.string,
+  round: PropTypes.bool,
+};
+
+Button.defaultProps = {
+  icon: '',
+  color: 'white',
+  round: false,
+};
```
Comment (PacoMojica) [inline src/components/Button/Button.component.jsx:28]: Great 👍  this is the first challenge where I see someone use the PropTypes

File: src/components/Button/styled.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  justify-content: center;
+`;
+
+const StyledIcon = styled(FontAwesomeIcon)`
+  color: #fff;
+  font-size: var(--font-size-lg);
+  margin-right: var(--spacing-md);
+`;
+
+export { StyledButton, StyledIcon };
```
Comment (PacoMojica) [inline src/components/Button/styled.js:21]: Good use of the styled-components, you can go one step further and have something like:

```jsx
export const StyledButton = styled.button`
   ...
`;

export const StyledIcon = styled(FontAwesomeIcon)`
  ...
`;
```
and delete line 21, in this case, you have a few components but sometimes you can have several components that you want to export in a file and the approach you used here is kind of hard to maintain

File: src/components/Card/Card.component.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <Banner url={thumbnail} alt="thumbnail" />
+      <FlexContainer padding={{ horizontal: 'md' }} column>
+        <Title size="xsm" color="black">
+          {title}
+        </Title>
+        <Text size="md" color="gray" margin={{ top: 'sm' }}>
+          {description}
+        </Text>
+      </FlexContainer>
+    </Container>
```
Comment (PacoMojica) [inline src/components/Card/Card.component.jsx:21]: Your card looks great, it's small and concise

File: src/components/utils/__test__/utils.test.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import utils from 'components/utils';
+
+describe('utils', () => {
+  describe('getSpacingCSS', () => {
+    it('should return empty string when the using obj is present', () => {
```
Comment (PacoMojica) [inline src/components/utils/__test__/utils.test.js:5]: This test is well defined, you cover a lot of edge cases, good stuff 👍

**Review by PacoMojica — COMMENTED** [review]
Hello, Victor, 👋  Your challenge contains all we required, in your next delivery I'd like to see a test created for any component you want, great work 👍, keep the good things coming
